addons/website/controllers/form.py

```diff
@@ -148,6 +152,16 @@ def extract_data(self, model, values):
         error_fields = []
         custom_fields = []
 
+        if 'description' in authorized_fields:
+            if request.session.uid:
+                user_name = request.env.user.name
+                form_email = values[self._get_email_field_name(dest_model._name)]
+                _authenticate_message = _("This %s was submitted by %s on behalf of %s") % (model.name, user_name, form_email)
```

Comment:
Better use named parameters, helps when translating as it gives context to people :)